src/Tribe/Venue.php

```diff
@@ -698,7 +698,12 @@ public static function get_fetch_callback( $event ) {
 		}
 
 		return static function () use ( $event ) {
-			$venue_ids = array_map( 'absint', (array) get_post_meta( $event, '_EventVenueID' ) );
+			$venue_ids = array_filter(
+				array_map(
+					'absint',
+					(array)get_post_meta( $event, '_EventVenueID' )
```

Comment:
Space after `(array)`
```suggestion
					(array) get_post_meta( $event, '_EventVenueID' )
```